src/styles.scss

```diff
@@ -658,8 +660,12 @@ $green-font-color: #19B028;
   margin-top: 2px;
 }
 
-.feed-create-post__image-icon {
-  margin-top: 2px;
+.feed-create-post__icon {
+	background-color: var(--secondary);
+	width: 34px;
+	height: 30px;
+	text-align: center;
+	line-height: 30px;
```

Comment:
this class isn't used anywhere. Let's remove and put back the `.feed-create-post__image-icon` class.